tests/test_mets.py

```diff
@@ -158,6 +159,46 @@ def test_fromfile_invalid_xlink_href(self):
             metsrw.METSDocument.fromfile(
                 'fixtures/mets_invalid_xlink_hrefs.xml')
 
+    def test__analyze_fptr(self):
+        parser = etree.XMLParser(remove_blank_text=True)
+        tree = etree.parse('fixtures/mets_dir_with_fptrs.xml', parser=parser)
+        mw = metsrw.METSDocument()
+
+        # Test that exception is raised when fileSec cannot be found.
+        fptr_elem = etree.fromstring('<fptr FILEID="12345"/>')
+        with pytest.raises(metsrw.exceptions.ParseError,
+                           match='12345 exists in structMap but not fileSec'):
+            mw._analyze_fptr(fptr_elem, tree, 'directory')
```

Comment:
staticmethods don't need an instance, so you should be able to call it like `metsrw.METSDocument._analyze_fptr` and skip the initialization if you like.